### PR TITLE
Update bundles

### PR DESCRIPTION
**Pull request by miurahr**
- Pick changes from manuel5.n branch

### CODE REVIEW

File: src/org/omegat/Bundle.properties [hunk 1]
```diff
@@ -245,46 +254,44 @@ ENCODING_AUTO=<auto>
...
 TF_MENU_FILE=&Project
 
 TF_MENU_FILE_CREATE=&New...
 
 TF_MENU_FILE_TEAM_CREATE=Download &Team Project...
 
 TF_MENU_FILE_OPEN=&Open...
-
-TF_MENU_FILE_OPEN_RECENT=Open Recent Pro&ject
+TF_MENU_FILE_OPEN_RECENT=Open &Recent...
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:284]: This changes shortcut (&j -> &R)

File: src/org/omegat/Bundle.properties [hunk 2]
```diff
@@ -245,46 +254,44 @@ ENCODING_AUTO=<auto>
...
 TF_MENU_FILE_CLEAR_RECENT=&Clear Menu
 
-TF_MENU_FILE_IMPORT=Cop&y Files to Source Folder...
-TF_FILE_IMPORT_TITLE=Copy Files to Source Folder...
-
-TF_MENU_WIKI_IMPORT=Download Media&Wiki Page...
+TF_MENU_FILE_IMPORT=Add Files...
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:287]: This removes shortcut (&y)

File: src/org/omegat/Bundle.properties [hunk 3]
```diff
@@ -295,76 +302,68 @@ TF_MENU_FILE_COMMIT=Commit Source Files
...
-
-TF_MENU_FILE_PROJWIN=Project &Files...
+WM_PROJECTMENU_FILEFILTERS=Local File &Filters...
+TF_MENU_FILE_PROJWIN=Source &Files...
 
 TF_MENU_FILE_ACCESS_PROJECT_FILES=&Access Project Contents
-
-TF_MENU_FILE_ACCESS_ROOT=&Root
-
+TF_MENU_FILE_ACCESS_ROOT=&Project Folder
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:316]: Shortcut change(&R->&P)

**Review by miurahr — COMMENTED** [review]
There are so many unintended Mnemonics changes.
An order of entries are changed, so it is difficult check changes.

File: src/org/omegat/Bundle.properties [hunk 4]
```diff
@@ -1269,18 +1284,18 @@ LATEXFILTER_FILTER_NAME=LaTeX
 
 POFILTER_FILTER_NAME=PO
 POFILTER_OPTIONS_TITLE=PO Filter Options
-POFILTER_ALLOW_BLANK=&Allow blank translations in the target files
-POFILTER_ALLOW_BLANK_SEGMENT=Allow translating &blank source segments
-POFILTER_SKIP_HEADER=&Skip PO header
-POFILTER_AUTO_FILL_IN_PLURAL_STATEMENT=Auto replace 'nplurals=INTEGER; &plural=EXPRESSION;' in header
+POFILTER_ALLOW_BLANK=&Allow blank target segments
+POFILTER_ALLOW_BLANK_SEGMENT=Translate &blank source segments
+POFILTER_SKIP_HEADER=&Ignore PO header
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:1289]: &S->&I

File: src/org/omegat/Bundle.properties [hunk 5]
```diff
@@ -1342,12 +1357,12 @@ RBFILTER_FILTER_NAME=Java(TM) Resource Bundles
 RBFH_ERROR_ILLEGAL_U_SEQUENCE=Encountered illegal \\u sequence
 RB_FILTER_OPTIONS_TITLE=Java Resource Bundles Filter Options
 RB_FILTER_REMOVE_STRINGS_UNTRANSLATED=&Remove untranslated strings in the target files
-RB_FILTER_NOT_CONVERT_UNICODE_CHARACTERS=&Do not unescape Unicode literals (\\uXXXX)
-RB_FILTER_SUPPORT_JAVA8_ENCODING=Force escaping Unicode literals as compatible to Java 8 to write
+RB_FILTER_NOT_CONVERT_UNICODE_CHARACTERS=Keep Unico&de literals (\\uXXXX)
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:1360]: Removal of &D

File: src/org/omegat/Bundle.properties [hunk 6]
```diff
@@ -1553,34 +1568,28 @@ LICENSEDIALOG_PREFACE=\
...
 
 # FiltersCustomizer.java
 
-FILTERSCUSTOMIZER_TITLE=File Filters
-FILTERSCUSTOMIZER_TITLE_PROJECTSPECIFIC=Project Specific File Filters
+FILTERSCUSTOMIZER_TITLE=Global File Filters
+FILTERSCUSTOMIZER_TITLE_PROJECTSPECIFIC=Local File Filters
 
-FILTERSCUSTOMIZER_CHECKBOX_PROJECTSPECIFIC=&Make the file filter settings project specific
+FILTERSCUSTOMIZER_CHECKBOX_PROJECTSPECIFIC=&Use local file filter settings
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:1590]: &M->&U

File: src/org/omegat/Bundle.properties [hunk 7]
```diff
@@ -1589,7 +1598,7 @@ FILTERSCUSTOMIZER_BUTTON_OPTIONS=O&ptions...
 FILTERSCUSTOMIZER_OPTION_GLOBAL_REMOVE_TAGS=Remove &leading and trailing tags
 FILTERSCUSTOMIZER_OPTION_GLOBAL_REMOVE_SPACES_NONSEG=Remove leading and trailing &whitespace in non-segmented projects
 FILTERSCUSTOMIZER_OPTION_GLOBAL_PRESERVE_SPACES=Preserve spaces &for all tags
-FILTERSCUSTOMIZER_OPTION_GLOBAL_IGNORE_FILE=&Ignore file context when identifying segments with alternate translations
+FILTERSCUSTOMIZER_OPTION_GLOBAL_IGNORE_FILE=&Do not use the file name to identify alternate translations
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:1601]: &I->&D

File: src/org/omegat/Bundle.properties [hunk 8]
```diff
@@ -1997,40 +2001,40 @@ SC_ADD_TO_DICTIONARY=Add To Dictionary
 GUI_SPELLCHECKER_TITLE=Spellchecker Setup
 PREFS_TITLE_SPELLCHECKER=Spellchecker
 
-GUI_SPELLCHECKER_AUTOSPELLCHECKCHECKBOX=&Automatically check the spelling of text
+GUI_SPELLCHECKER_AUTOSPELLCHECKCHECKBOX=&Spellcheck the translation
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:2004]: &A->&S

File: src/org/omegat/Bundle.properties [hunk 9]
```diff
@@ -1997,40 +2001,40 @@ SC_ADD_TO_DICTIONARY=Add To Dictionary
...
 GUI_SPELLCHECKER_FILE_CHOOSER_TITLE=Dictionary File Folder
 
 GUI_SPELLCHECKER_DIRECTORYCHOOSERBUTTON=&Browse...
 
-GUI_SPELLCHECKER_INSTALLBUTTON=&Install new dictionary...
+GUI_SPELLCHECKER_INSTALLBUTTON=&Install new language...
 
-GUI_SPELLCHECKER_UNINSTALLBUTTON=&Remove
-GUI_SPELLCHECKER_URL_LABEL=URL of online dictionaries:
+GUI_SPELLCHECKER_UNINSTALLBUTTON=&Delete
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:2016]: &R->&D

**Review by miurahr — COMMENTED** [review]
Further changes on mnemonics

**Comment by brandelune** [comment]
Thank you for checking all that.

**Pull request closed without merging**